### DISCUSSION TITLE
Comparison of Apache Cloudstack security and SEV-SNP technology.

### DISCUSSION THREAD

**Discussion by miloserdoff** [apache/cloudstack#10141] [Q&A]
##### ISSUE TYPE

 * Other

##### COMPONENT NAME

~~~
Data Security
~~~

##### CLOUDSTACK VERSION

~~~
4.18.2.5
~~~

##### CONFIGURATION

No

##### OS / ENVIRONMENT

Ubuntu 18.04

##### SUMMARY

I want to compare the security provided by Apache Cloudstack and AMD SEV-SNP technology. What can Cloudstack do that SEV-SNP cannot?

##### EXPECTED RESULTS

~~~
Describe in brief all the benefits of data encryption and protection of all information provided by the Apache Cloudstack platform. What are the advantages of Cloudstack compared to AMD SEV-SNP? What can Cloudstack do that AMD SEV-SNP cannot provide?
~~~

**Comment by DaanHoogland** [comment]
@miloserdoff , this sounds like you are asking for a document on the comparison. There is none available, so you'll have to find a volunteer to write it.

**Comment by pcfriek1987** [comment]
Not sure what kind of comparison that would be either.. as Cloudstack is software, SEV-SNP a CPU feature.

**Comment by DaanHoogland** [comment]
closed this for lack of interaction, please reopen or open a new item if applicable
